include/functions.h

```diff
@@ -9,7 +9,7 @@ void osCleanupThread(void);
 
 s32 heap_malloc(s32 size);
 HeapNode* _heap_create(void* addr, s32 size);
-void dma_copy(void* romStart, void* romEnd, void* vramDest);
+s32 dma_copy(s32 romStart, s32 romEnd, void* vramDest);
```

Comment:
If we're doing this, we should also modify ld_addrs.h extern generation to reflect ROM addresses as s32 rather than void*.